jsapp/js/components/languages/languagesListStore.ts

```diff
@@ -56,7 +56,7 @@ export default class LanguagesListStore {
 
   private onAnyFail(response: FailResponse) {
     this.isLoading = false;
-    notify(response.responseText, 'error');
+    notify(response.responseText || t('Unknown error'), 'error');
```

Comment:
If we have the status code for these, I think I want to add them to the message like you did with the generic handler, because it could help in situations where someone takes a screenshot and sends it to the support team.

Reply:
Could we use utils.ts `handleApiFail(FailResponse)` here?